radionice/srednji/lekcije/030_kauzativ/030_kauzativ.tex

```diff
@@ -32,39 +39,85 @@
 	
 	Ova glagolska forma se koristi u dva konteksta:
 
-	1) kada se tjera nekoga nešto učiniti
-	2) kada se dopušta nešto nekome učiniti
+	1) kada se želi natjerati nekoga da nešto učini
+	
+	2) kada se želi dopustiti nekome da nešto učini
 	
 	\begin{reibun}
 		\rei{先生が学生に宿題をたくさんさせた。}{ Učitelj je natjerao učenike da pišu puno zadaće.}
-		\rei{先生が質問をたくさん聞かせてくれた。}{Učitelj je dopustio (nekome) da pita puno pitanja.}
+		\rei{先生が質問をたくさん聞かせてくれた。}{ Učitelj je dopustio (nekome) da pita puno pitanja.}
 		\rei{今日は仕事を休ませてください。}{ Dopustite mi da se danas odmorim od posla.}
 		\rei{その部長は、よく長時間働かせる。}{ Voditelj odjela često tjera (zaposlenike) da rade puno sati.}
 	\end{reibun}
 	
-	\fukudai{Vježba}
+	\newpage
+	\dai{Kauzativ pasiv}
+	
+	\fukudai{Tvorba}
```

Comment:
Gledajući obje tablice, pitam se jesu li možda suvišne, to jest možemo li ih komprimirati u tablice koje sadrže nepravilnosti + opise pravilnosti.